docs/core/compendium_de.md

```diff
@@ -0,0 +1,1177 @@
+optional.
+
+Veraltet: verwenden Sie stattdessen [HealthChecks](#healthchecks).
+
+### HealthChecks
+
+HealthChecks definieren eine Liste von [HealthCheck](#type-healthcheck). Sie bieten mehrere Möglichkeiten, den Zustand
+des Dogus auf Beobachtbarkeit zu prüfen. Dieses Feld ist optional.
+
+HealthChecks werden in verschiedenen Anwendungsfällen verwendet:
+
+- um die `dogu is starting` Seite anzuzeigen, bis das Dogu beim Start healthy ist
+- zur Überwachung über `cesapp healthy <dogu-name>`
+- für die Überwachung über das Admin-Dogu
+- um zu vermeiden, dass ein unhealthy Dogu gesichert wird
+
+Es gibt verschiedene Arten von HealthChecks:
+
+- state, über den `/state/<dogu>` Registry-Schlüssel, der von ces-setup gesetzt wird
+- tcp, um auf einen offenen Port zu prüfen
+- http, um einen Statuscode einer http-Antwort zu prüfen
+
+Sie müssen ohne Authentifizierung ausführbar sein.
+
+Beispiel:
+
+```
+"HealthChecks": [
+  {"Type": "state"}
+  {"Type": "tcp",  "Port": 8080},
+  {"Type": "http", "Port": 8080, "Path": "/my/health/path"},
+]
+```
+
+### ServiceAccounts
+
+ServiceAccounts enthält eine Liste von [ServiceAccount](#type-serviceaccount). Dieses Feld ist optional.
+
+Ein Service-Account schützt sensible Daten, die für ein anderes Dogus verwendet werden. Service-Account-Konsumenten
+werden von Service-Account-Produzenten unterschieden. Um ein Servicekonto zu erzeugen, muss ein Dogu die Skripte
+service-account-create und service-account-delete in [core.ExposedCommands](#exposedcommands) implementieren.
+Um ein
+Service-Account zu konsumieren, sollte ein Dogus eine Anfrage in ServiceAccounts definieren.
+
+Bei der Installation erkennt ein Dogu-Client eine Service-Account-Anfrage und führt das Skript "service-account-create"
+mit den Informationen des Service-Account-Produzenten aus. Die Anmeldedaten werden dann in der Cloudogu
+EcoSystem-Registry gespeichert.
+
+Beispielpfade:
+
+- config/redmine/sa-postgresql/username
+- config/redmine/sa-postgresql/password
+- config/redmine/sa-postgresql/database
+
+Das Dogu selbst kann dann den Service-Account verwenden, indem er die Anmeldeinformationen mit `doguctl` im Startskript
+liest und entschlüsselt.
+
+Beispiele:
+
+```
+"ServiceAccounts": [
+  {
+    "Type": "ldap",
+    "Params": [
+      "rw"
+    ]
+  },
+  {
+   "Type": "k8s-dogu-operator",
+   "Kind": "k8s"
+  }
+],
+```
+
+### Privileged
+
+Privileged gibt an, ob der Docker-Socket in das Container-Dateisystem eingehängt werden soll. Dieses Feld ist optional.
+Der Standardwert ist `false`.
+
+Aus Sicherheitsgründen ist es sehr empfehlenswert, Privileged auf false zu setzen, da fast kein Dogu rückwirkend
+Einblick in den Container erhalten sollte.
+
+Beispiel:
+
+- false
+
+### Configuration
+
+Configuration enthält eine Liste von [ConfigurationField](#type-configurationfield). Dieses Feld ist optional.
+
+Es beschreibt allgemeine Eigenschaften des Dogus in der Cloudogu EcoSystem-Registry.
```

Comment:
We should choose a spelling style for Cloudogu EcoSystem[-]Registry. We seem to have versions regarding the `-`.